packages/grid/package.json

```diff
@@ -21,7 +21,7 @@
   "sideEffects": false,
   "types": "dist/typings/index.d.ts",
   "dependencies": {
-    "@zendeskgarden/container-splitter": "^2.0.0",
+    "@zendeskgarden/container-splitter": "^2.0.2",
```

Comment:
Will be bumped to `^2.0.3` pending https://github.com/zendeskgarden/react-containers/pull/488